docs/src/whatsnew/3.7.rst

```diff
@@ -1,7 +1,7 @@
 .. include:: ../common_links.inc
 
-v3.7 (16 Aug 2023) [release candidate]
-**************************************
+v3.7 (16 Aug 2023)
```

Comment:
As mentioned in #5454 this should be 31 Aug. 16 Aug was the date of the release candidate